UNRELEASED.md

```diff
@@ -10,6 +10,11 @@
          * Some bug fix, see #124
 
      DO NOT LEAVE A BLANK LINE BELOW THIS PREAMBLE -->
+### Breaking changes
+
+ * Dedicated exit codes for type-checking errors, parser errors, and evaluation
+   errors (e.g., due to unsupported language constructs or operators), see #1749
```

Comment:
```suggestion
 * Introduced dedicated exit codes for type-checking errors, parser errors, and evaluation errors (e.g., due to unsupported language constructs or operators), see #1749
```

Reply:
Good point!
Not sure if I prefer past tense indicative or imperative with these though 🤣